features/google/src/model/importers/activities-importer.js

```diff
@@ -1,9 +1,11 @@
 import UserActivity from "../entities/user-activity";
 
-const activityRegex = /\/My Activity\/.*\.html$/;
+const activityHtmlRegex = /\/My Activity\/.*\.html$/;
```

Comment:
This is "Mi actividad" in my Spanish takeout. Again, possibly German should be taken into account here.